api/api.go

```diff
@@ -219,7 +219,7 @@ func (t *TLSConfig) Copy() *TLSConfig {
 	return nt
 }
 
-func defaultHttpClient() *http.Client {
+func DefaultHttpClient() *http.Client {
```

Comment:
This function is pretty small and isn't really nomad api specific.  I'd probably suggest copying it over, rather than exposing and committing to it as part of our api nomad interface?

Reply:
fixed